### PR TITLE
Extended Position Schema Upgrade Aave forks

### PR DESCRIPTION
**Pull request by dmelotik**
Deliverables:
- Add extended schema to aave v2 forks in a new directory
- Fix `handleCollateralEnabled/Disabled` this event is actually for a position not the whole market

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ This repo contains subgraphs defined using a set of standardized schemas. These
...
+| [Aave v2](https://aave.com/) | 🛠 | 2.0.1 / 1.2.2 / 1.0.0 | [![Aave V2 Ethereum](./docs/images/chains/ethereum.png)](https://thegraph.com/hosted-service/subgraph/messari/aave-v2-ethereum) [![Aave V2 Ethereum ARC](./docs/images/chains/ethereum.png)](https://thegraph.com/hosted-service/subgraph/messari/aave-v2-ethereum-arc) [![Aave V2 Ethereum RWA](./docs/images/chains/ethereum.png)](https://thegraph.com/hosted-service/subgraph/messari/aave-v2-ethereum-rwa) [![Aave V2 Ethereum AMM](./docs/images/chains/ethereum.png)](https://thegraph.com/hosted-service/subgraph/messari/aave-v2-ethereum-amm) [![Aave V2 Avalanche](./docs/images/chains/avalanche.png)](https://thegraph.com/hosted-service/subgraph/messari/aave-v2-avalanche) [![Aave V2 Polygon](./docs/images/chains/matic.png)](https://thegraph.com/hosted-service/subgraph/messari/aave-v2-polygons) |
```
Comment (this-username-is-taken) [inline README.md:92]: We don't need to keep track of extended schema. These are for experimental purposes and won't be used in production

File: README.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ This repo contains subgraphs defined using a set of standardized schemas. These
...
 | [dForce](https://dforce.network) | 🛠 | 1.3.0 / 1.0.1 / 1.0.0 | [![dforce Arbitrum](./docs/images/chains/arbitrum.png)]() [![dforce Avalanche](./docs/images/chains/avalanche.png)]() [![dforce BSC](./docs/images/chains/bsc.png)]() [![dforce Ethereum](./docs/images/chains/ethereum.png)]() [![dforce Optimism](./docs/images/chains/optimism.png)]() [![dforce Polygon](./docs/images/chains/matic.png)]() |
 | Euler Finance | 🔨 | | |
-| [Geist](https://geist.finance/markets) | 🛠 | 1.3.0 / 1.0.0 / 1.0.0 | [![Geist Protocl](./docs/images/chains/fantom.png)](https://thegraph.com/hosted-service/subgraph/messari/geist-fantom) |
+| [Geist](https://geist.finance/markets) | 🛠 | 2.0.1 / 1.0.0 / 1.0.0 | [![Geist Protocl](./docs/images/chains/fantom.png)](https://thegraph.com/hosted-service/subgraph/messari/geist-fantom) |
```
Comment (this-username-is-taken) [inline README.md:102]: No need to extend schema for Geist. Also no need for ARC and RWA

Comment (this-username-is-taken) [reply README.md:102]: Consider this a separate subgraph from `aave-v2`

Comment (dmelotik) [reply README.md:102]: sounds good

**Comment by dmelotik** [comment]
Another rebase gone wrong 😔

**Comment by dmelotik** [comment]
closing and reopening bc it's much easier than trying to fix this

**Pull request closed without merging**